apps/dotcom/client/package.json

```diff
@@ -20,10 +20,11 @@
 		"test": "yarn run -T jest",
 		"i18n:extract": "formatjs extract 'src/{tla,pages,components}/**/*.ts*' --out-file public/tla/locales/en.json --additional-component-names F --id-interpolation-pattern '[md5:contenthash:hex:10]' --format lokalise",
 		"i18n:compile": "formatjs compile-folder public/tla/locales public/tla/locales-compiled --format lokalise",
-		"e2e": "rm -rf 'e2e/.auth' && yarn playwright test",
-		"e2e-x10": "yarn playwright test --repeat-each=10",
-		"e2e-debug": "yarn playwright test --debug",
-		"e2e-ui": "yarn playwright test --ui"
+		"e2e": "rm -rf 'e2e/.auth' && yarn playwright test --project=chromium",
```

Comment:
Had to scope these to a specific project, so that we don't run the staging tests as well.